src/app/dashboard/plugin-config.html

```diff
@@ -1,13 +1,15 @@
-<md-dialog>
+<md-dialog flex="30">
   <md-dialog-content>
-    <!-- {{vm.plugin.title}} -->
+    <h2>{{vm.plugin.title}}</h2>
```

Comment:
Yeah, that's probably a good idea. I don't think we save that currently, so it'll take a little re-working.